addons/mail/controllers/discuss/channel.py

```diff
@@ -48,7 +48,7 @@ def discuss_channel_info(self, channel_id):
 
     @http.route("/discuss/channel/messages", methods=["POST"], type="jsonrpc", auth="public")
     @add_guest_to_context
-    def discuss_channel_messages(self, channel_id, search_term=None, before=None, after=None, limit=30, around=None):
+    def discuss_channel_messages(self, channel_id, fetch_params=None):
```

Comment:
Do we still need to support `fetch_params` default `None` in all of those routes?

Each individual params needed to be `None` because they were not all required at the same time, but I think at least one `fetch_params` should always be given.

Reply:
Not a good idea to not have limit.

What tests are we talking about? We might be better off adding a relevant param (limit?) in the tests rather than supporting None in all cases when it's never used in practice, especially if we need to add {} everywhere to compensate.